docs/concepts/capabilities.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: 'Capabilities'
+icon: 'ti-anchor' # themify icon pack : https://themify.me/themify-icons
+description: 'Application declaration, deployment, and management'
+sidebar_position: 1
+type: 'docs'
+---
+
+## Overview
+
+A ***capability** is a pluggable piece of functionality for an application. More specifically, capabilities are common, abstracted functionalities&mdash;usually the sort of utilities that aren't core business logic for a given application, such as...
```

Comment:
```suggestion
A ***capability** is a reusable, pluggable piece of functionality. 

Applications use capabilities while implementing business logic&mdash; to do things like store and retrieve data, access secrets, and more. Here are some examples:
```

What do you think about this? I think that first line could definitely stand on it's own... but I'm not sure how else to make the second line fit in!